src/Calculator/Views/MainPage.xaml.cpp

```diff
@@ -128,6 +136,7 @@ void MainPage::OnAppPropertyChanged(_In_ Platform::Object ^ sender, _In_ Windows
         if (newValue == ViewMode::Standard)
         {
             EnsureCalculator();
+            EnsureAlwaysOnTopCalculator();
```

Comment:
We shouldn't need to call this every time we call `EnsureCalculator()`. We should only load Always On Top when that button is clicked. Otherwise, we're doing a lot of extra work for no reason.